pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -354,9 +359,12 @@ public synchronized void buildRouting(String tableNameWithType) {
       }
     }
 
+    QueryConfig queryConfig = tableConfig.getQueryConfig();
+    Long timeoutMs = queryConfig != null ? queryConfig.getTimeoutMs() : null;
```

Comment:
I saw there are two places which use different types of long. Can we unify them together? E.g. always use -1 to denote `getTimeoutMs` is null.

Reply:
For json ser-de reason, it is risky to put `-1` for `null` value (serialize & deserialize will break). I'll just use `null` in both places then